bin/jsoo_common.ml

```diff
@@ -86,7 +85,11 @@ module Reason = struct
     (* you can't throw an Error here. jsoo parses the string and turns it
        into something else *)
     let throwAnything = Js.Unsafe.js_expr "function(a) {throw a}" in
-    try code |> Js.to_string |> Lexing.from_string |> f
+    let code =
+      (* Add ending new line as otherwise reason parser chokes with inputs such as "//" *)
```

Comment:
isn't this https://github.com/reasonml/reason/issues/2350, which was fixed?

Reply:
It looks like. I have no idea why it's reappearing now, maybe because in playground case we don't have EOF?